modules/installation-network-user-infra.adoc

```diff
@@ -193,7 +193,7 @@ endif::ibm-z-kvm[]
 
 .4+|TCP
 |`1936`
-|Metrics
+|Metrics.
```

Comment:
I'm not sure if I'd add a period to this or not. The only other ones in the table that have periods are complete sentences. If we keep the period we might want to add a period for the following, and maybe others (seems like a can of worms):

**Network reachability tests**

**The default ports that Kubernetes reserves**

**Kubernetes node port**

Reply:
Thanks, @stevsmit . Valid point. I removed the period.